Design/Workbench/design-development.md

```diff
@@ -0,0 +1,22 @@
+# Development
+
+## Coding Style
+
+It is proposed we introduce a new Python style guide for Mantid. All new and refactored code should then follow this guide. The [IPython style guide][ipython-style] offers quite nice guidelines and it is suggested that we
+follow these guidelines where applicable, including a template file for new code that could be used in the `class_maker` to generate new files.
+
+## Static Analysis
+
+We will no longer be able to rely on a compiler to catch simply errors - it is therefore imperative that we use static analysis to its fullest extent. While `flake8` is good it is not as thorough as `pylint` and given our
```

Comment:
Flake8 is also a lot faster, is the benefit of pylint worth the speed penalty?

Also I'm not too happy with the idea of Flake8 on some code and PyLint elsewhere (I could be reading this wrong though).